SCClassLibrary/Common/Quarks/QuarksGui.sc

```diff
@@ -620,13 +636,14 @@ QuarkRowView {
 
 		isInstalled = quark.isInstalled;
 		btn.value = isInstalled.binaryValue;
-
 		// this column has invisible text
 		// its used to sort the rows so that installed quarks are at the top
 		treeItem.setString(0, isInstalled.if("Y", { quark.isDownloaded.if("N", "") }));
 		// 1 is the install button. its not possible to sort by this column
 		treeItem.setString(2, quark.name ? "");
 		treeItem.setString(3, (quark.version ? "").asString);
-		treeItem.setString(4, (quark.summary ? "").replace(Char.nl.asString," ").replace(Char.tab.asString, ""));
+		treeItem.setString(4, (quark.lastUpdated ? "").asString);
+		treeItem.setString(5, (quark.author ? "").asString);
+		treeItem.setString(6, (quark.summary ? "").replace(Char.nl.asString," ").replace(Char.tab.asString, ""));
```

Comment:
the `asStrings` here are not necessary -- `replace` takes Chars and Strings for either argument. mind removing them?